### PR TITLE
Rewrite logging and configuration

### PR DESCRIPTION
**Pull request by fredrikhl**
Refactors the `zabbix_cli.config` module:

- The `configuration` class is rewritten to simply wrap the `ConfigParser` object directly.
- Logic for reading the configuration file and representing the configuration values are now separated.

Refactors the `zabbix_cli.logs` module:

- Removes the custom logger class
- Adds log formatters and filters to support custom log record fields
- Adds proper logging module configuration, to support multiple loggers and use the logger hierarchy better.

In addition, the scripts have been updated to support these changes:

- Read config using functions from `zabbix_cli.config`
- Add username to log messages using the `zabbix_cli.logs.LogContext` class
- Use module level loggers for all logging, and apply handlers to the root logger
- Remove if/else blocks for logging - logging always happens, but we use a NullHandler if logging is disabled in config.

### CODE REVIEW

File: zabbix_cli/cli.py [hunk 1]
```diff
@@ -38,12 +41,13 @@
 import textwrap
 import time
 
-import ipaddr
-
 import zabbix_cli
+import zabbix_cli.version
```
Comment (paalbra) [inline zabbix_cli/cli.py]: The version module has been removed. I guess the import got erroneously merged back in?

Comment (paalbra) [reply zabbix_cli/cli.py]: zabbix-cli now crashes with: `ModuleNotFoundError: No module named 'zabbix_cli.version'`

Comment (fredrikhl) [reply zabbix_cli/cli.py]: Probably a merge-issue. Fixed!

**Comment by paalbra** [comment]
Looks good to me

**Pull request merged**